lib/conjur/config.rb

```diff
@@ -0,0 +1,39 @@
+module Conjur
+  module Config
```

Comment:
Have you checked if this all actually works? I seem to remember there was a reason why I didn't split it all in separate libs. Puppet does some bizarre stuff when loading code, trying to keep modules separated. Maybe it's not as bad anymore; if that's the case, great :)

Reply:
As I mentioned in Slack, this has been working for me so far. But the fact that you've seen issues before should be a risk we test further.

Do you have any ideas on test contexts that might exhibit this? The main idea I have is to go back to earlier Puppet versions. It looks like I'm testing with `5.5.14`.

Reply:
I honestly don't remember :) I mean, if it works, then great. But let's just keep it in mind if something breaks ;)